akka-docs/rst/scala/http/routing-dsl/directives/debugging-directives/logRequestResult.rst

```diff
@@ -30,3 +30,13 @@ Example
 
 .. includecode2:: ../../../../code/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala
    :snippet: logRequestResult
+
+
+LongerExample : Logging the request and the response time of every request.
+-------------
```

Comment:
underline should be as long as the text.
text is too long though.

```
Longer example
----------
```

And below explain what it is